modules/oe_webtools_laco_widget/src/Form/WebtoolsLacoWidgetSettingsForm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_laco_widget\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Settings form for module.
+ */
+class WebtoolsLacoWidgetSettingsForm extends ConfigFormBase {
+
+  /**
+   * Name of the config being edited.
+   */
+  const CONFIGNAME = 'oe_webtools_laco_widget.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
```

Comment:
Since we have strict types, is it not required to state the return type of the function?

Reply:
I don't think so and it's a common practice to use {@inheritdoc} even with strict types. I see it in many places, here's one:
https://github.com/openeuropa/oe_webtools/blob/master/modules/oe_webtools_laco_widget/tests/src/Functional/LacoWidgetTest.php